hub/compression.py

```diff
@@ -55,7 +56,12 @@
     _compression_types[c] = IMAGE_COMPRESSION
 for c in BYTE_COMPRESSIONS:
     _compression_types[c] = BYTE_COMPRESSION
+for c in AUDIO_COMPRESSIONS:
+    _compression_types[c] = AUDIO_COMPRESSION
 
 
 def get_compression_type(c):
-    return _compression_types[c]
+    ret = _compression_types.get(c)
```

Comment:
I like the change that you made here. Using `.get()` method on a dictionary doesn't raise the exception when the specified key is not present.

Reply:
i think we do definitely want an exception to be raised here

Reply:
Its not a simple dict access, there are some edge cases.